testing/environments/docker/kafka/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM spotify/kafka
```

Comment:
I'm undecided if these files belong into libbeat or here in the "general" environment area. Reason for keeping it in libbeat is as they are only used there. Reason for having it here is that our "manual testing" setup should also have kafka available. Same for redis.